README.md

```diff
@@ -10,6 +10,37 @@ If this is your first time here, try [experimenting with the live demo](http://b
 Installation
 ------------
 
+## With Vagrant
+
+    $ vagrant up
```

Comment:
Include installation instructions and instructions on what to do after this command, i.e. `Now browse to http://localhost:8080 to see the OAuth2 Demo Application`

Reply:
Ah, yes.  My bad.  Will fix.

Reply:
Hmm.  I'm having none of these errors.   Can you give me some instructions on how to recreate?

Reply:
Oh.  You're running through tests.  Yeah.  Should've done that.

Reply:
I don't even see where in the code you're connecting to 8081.

Reply:
Ok.  Figured it out.  Now I'm getting the connection error.  Fixing ...